README.md

```diff
@@ -26,5 +33,10 @@ colored_circles.index_to_circle(1234)
 # We could now use this to make an svg of a circle.
 ```
 
-This repository uses [famed](https://famed.morphysm.com/boards/0kok0/emojihash) to reward contributions. Visit https://www.famed.morphysm.com to learn more about how to get famed or get in touch: contact@morphysm.com
+# Parameter analysis
```

Comment:
Please make split the pdf into two parts, parameter analysis (page 3) and before that as a subsection of the colored circles pages 1 and 2. You can call the subsection "High level description of the algorithm". Thx.